vulkan selection finished

richardlayman opened the pull request:
All selection items for vulkan are finished. This means that it's ready for implementation and has been tested. I can now move on to adding Vulkan into Qt which will be the last major hurdle for v0.1.

mottosso commented:
Exciting!

I'll be having another crack at getting this running on Windows again today; won't be expecting much, but I really want to be working alongside you on this and I'm levelling up on C++ each day.

richardlayman commented:
I've removed the viewport to make way for the Vulkan viewport, so there is currently no 3d display. Also, to make the code as simple as possible, I removed all the Windows related items from Vulkan so they would need to be put back in. Once I get Vulkan into Qt and get the file format finished, I may start building packages for Arch. There are also tools to build for Fedora, Ubuntu and Debian, so I might try those as well.
